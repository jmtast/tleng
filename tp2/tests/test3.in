// Error de sintaxis: Voz sin comp√°s
#tempo negra 120
#compas 2/4

const oct1 = 5;

voz (0)
{
}
