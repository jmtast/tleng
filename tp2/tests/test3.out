Error:
Error de sintaxis en l√≠nea 9
